cmd/commands/server/factory.go

```diff
@@ -58,6 +59,10 @@ func NewCommandWith(
 		locationDetector = location.NewDetectorFake("")
 	}
 
+	// (Re)generate required security primitives before openvpn start
```

Comment:
This factory having certificate generation inside is a bit unexpected - maybe this can be done in the command itself? We can initialize `openVPNPrimitives` here and pass it to `Command` as a parameter for dependency injection - this will make it more transparent.